.storybook/preview-head.html

```diff
@@ -2,3 +2,4 @@
 <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.6.0/slick.min.css" />
 <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.6.0/slick-theme.min.css"
 />
+<script id="stripe-js" src="https://js.stripe.com/v3/" async></script>
```

Comment:
We'll need to make sure this dependency is being correctly pulled into force.

Reply:
Might be possible to use react-helmet for that, depending on SSR needs/compatibility.